CMakeLists.txt

```diff
@@ -78,15 +83,13 @@ if (CMAKE_VERSION VERSION_LESS 3.1 OR NOT USE_COMPILE_FEATURES)
   endif ()
 endif ()
 
-set (CMAKE_CXX_FLAGS "-O3 -g -mavx -mfma ${CMAKE_CXX_FLAGS}")
-
 if (BUILD_PYTHON)
   find_package (PythonInterp)
   find_package (PythonLibs)
   find_package (NumPy)
 
   # python site dir
-  if (PYTHON_EXECUTABLE AND NOT DEFINED PYTHON_SITE_DIR)
+  if (PYTHON_EXECUTABLE AND NOT DEFINED PYTHON_SITE_PACKAGES)

```

Comment:
Why are you changing the PYTHON_EXECUTABLE? #161 did not touch these lines..

Reply:
just a rename, I should have set this in a separate commit